packages/dex2jar.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,10 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'd2j-dex2jar'
+$category = 'Java'
```

Comment:
At the moment, apktool is the only tool installed in the `Android` category and Bytecode Viewer the only one in the `Java` category. I think it could make sense to merge these two categories into a `Java/Android`. Opinions?

Reply:
I moved this discussion to https://github.com/mandiant/VM-Packages/issues/671